.github/workflows/upload.yml

```
@@ -4,25 +4,33 @@ on:
     types: [published]
 
 jobs:
+  determine_runner:
+    name: Change to Large Runner
+    uses: ./.github/workflows/determine-workflow-runner.yml
+    with:
+      default_runner: ubuntu-latest
+      force_large_runner: true
   tests:
```

Just adding whitespace
```suggestion
      force_large_runner: true

  tests:
```